README.md

```diff
@@ -148,8 +148,9 @@ const Gitlab = require('gitlab/dist/es5').default
 
 // Instantiating
 const api = new Gitlab({
-  url:   'http://example.com', // Defaults to https://gitlab.com
-  token: 'abcdefghij123456'	// Can be created in your profile.
+  url:   'http://example.com',  // Defaults to https://gitlab.com
+  token: 'abcdefghij123456',    // Can be created in your profile.
+  rejectUnauthorized: false     // Should insecure certificates be rejected (true by default)
```

Comment:
Please document it separately away from Instantiating example and include mention of [`NODE_EXTRA_CA_CERTS`](nodejs.org/api/cli.html#cli_node_extra_ca_certs_file)

Reply:
Created a whole new section for describing HTTPS handling. How's that?

Reply:
Thanks!